content/en/ecosystem/distributions.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Distributions
+description:
+  List of OpenTelemetry distributions maintained by third parties.
+aliases: [/distributions]
```

Comment:
```suggestion
```

Reply:
Hmm, this was marked as resolved, but the alias is still there.

Reply:
Hmm, let me check that

Reply:
Ah, now: https://github.com/open-telemetry/opentelemetry.io/pull/3854/commits/27604c5b125bad84c4b55f13aa29a929f6e63405